cetmix_tower_server/models/cx_tower_scheduled_task.py

```
@@ -0,0 +1,246 @@
+        available_tasks.write({"is_running": True})
+
+        # WARNING: Explicit commit.
+        # Reason: We must atomically reserve scheduled tasks by setting is_running=True,
+        # so that only one cron worker processes each task, even if multiple workers
+        # pick up the cron job at the same time. Without this commit, the change would
+        # not be visible to other transactions until the end of the cron, leading to a
+        # race condition and possible double execution. Committing here immediately
+        # persists the reservation and prevents concurrent processing of the same task.
+        # NOTE: Explicit commits are generally discouraged in Odoo business logic and
+        # should only be used with clear justification.
+        self.env.cr.commit()  # pylint: disable=invalid-commit
```

Pylint (3.3.7)

[warning] Unknown option value for 'disable', expected a valid pylint message and got 'invalid-commit'

(W0012)